python/Dockerfile

```diff
@@ -17,16 +17,12 @@ RUN python --version
 
 WORKDIR /usr/src/app
 
-COPY its-client .
-COPY its-client.cfg .
+COPY its_client its_client
 COPY requirements.txt .
 COPY setup.cfg .
 COPY setup.py .
 
-RUN pip install wheel && \
-    python -m unittest && \
```

Comment:
Running unittest does not work from a clean tree:

```
ERROR: its_client (unittest.loader._FailedTest)
----------------------------------------------------------------------
ImportError: Failed to import test module: its_client
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/unittest/loader.py", line 470, in _find_test_path
    package = self._get_module_from_name(name)
  File "/usr/local/lib/python3.8/unittest/loader.py", line 377, in _get_module_from_name
    __import__(name)
  File "/usr/src/app/its_client/__init__.py", line 13, in <module>
    import its_client.quadtree
  File "/usr/src/app/its_client/quadtree.py", line 11, in <module>
    from pygeotile.tile import Tile
ModuleNotFoundError: No module named 'pygeotile'


----------------------------------------------------------------------
Ran 1 test in 0.000s

FAILED (errors=1)
```

Note: when you try and build the docker image locally, be sure to do so from a clean tree, without the its-client already built! Otherwise, the docker context may get build artefacts that make the Dockerfile work.

Reply:
And now that I did read the **entire** error message, I tihink I get the issue... Damn...